schema/resources/assertion.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "arxiv-external-links",
+    "description": "Clearinghouse for relations between arXiv e-prints and external resources",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+      "@type": {
+        "type": "string"
+      },
+      "ArXivID": {
```

Comment:
Let's use snake-case for keys (consistent with other arXiv projects). E.g. ``arxiv_id``

Reply:
0f64e45